cardano-testnet/src/Testnet/SubmitApi.hs

```diff
@@ -0,0 +1,140 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+{-# OPTIONS_GHC -Wno-unused-imports -Wno-unused-local-binds -Wno-unused-matches #-}
+
+module Testnet.SubmitApi
+  ( submitApi
+  , submitApiSubmitTransaction
+  ) where
+
+import           Control.Exception (IOException, handle)
+import           Control.Monad
+import           Control.Monad.IO.Class
+import           Data.Aeson (FromJSON (..), Value, (.=), (.:))
+import           Data.Bool
+import           Data.Eq
+import           Data.Either
+import           Data.Function
+import           Data.Functor
+import           Data.Int
+import           Data.IORef
+import           Data.Maybe
+import           Data.Ord
+import           Data.Semigroup
+import           Data.String
+import           Data.Text (Text)
+import           GHC.Generics
+import           GHC.Num
+import           Hedgehog (MonadTest)
+import           Hedgehog.Extras.Stock.Aeson (rewriteObject)
+import           Hedgehog.Extras.Stock.IO.Network.Sprocket (Sprocket (..))
+import           Hedgehog.Extras.Stock.Time
+import           System.FilePath.Posix ((</>))
+import           System.IO (FilePath, IO)
+import           Text.Show
+
+import qualified Data.Aeson as J
+import qualified Data.ByteString.Base16 as Base16
+import qualified Data.ByteString as BSS
+import qualified Data.ByteString.Lazy as LBS
+import qualified Data.ByteString.Char8 as BS8
+import qualified Data.HashMap.Lazy as HM
+import qualified Data.List as L
+import qualified Data.Text as T
```

Comment:
There are only 26 letters in the alphabet, so using single letters for qualified imports may mean we run out of letters. I would suggest `List` and `Text` for these.

Reply:
Fixed!